### PR TITLE
Full site

### PR DESCRIPTION
**Pull request by fobadara**
#full site
In this version I:
*  improved the styling of the landing page
* created a work section
* created a form section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,30 +8,236 @@
...
-                <address>
-                    <ul class="container nav-container">
-                        <li class="nav-link-container"><a href="github.com/fobadara" class="nav-link"><img src="images/github.svg" alt="An image showing Github's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/linkedin-icon.svg" alt="An image showing Linkedin's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/angellist.svg" alt="An image showing angelist's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/twitter.svg" alt="An image showing twitter's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/medium.svg" alt="An image showing medium's logo"></a></li>
+                </div>
+                <div class="btn-container" id="resume-btn-container">
+                    <button type="button" class="btn" id="resume-btn">Get My Resume</button>
```
Comment (Shaher-11) [inline index.html:160]: Increase the button width to get the look below
right

File: index.html [hunk 2]
```diff
@@ -8,30 +8,236 @@
...
+                    </div>
+                    <div class="header-container">
+                        <h3 class="header" id="lang">Languages</h3>
+                    </div>
+                    <ul class="container code-skill" id="code-lang-list">
+                        <li class="lang">Javascript</li>
+                        <li class="lang">Ruby</li>
+                        <li class="lang">Html</li>
+                        <li class="lang">css</li>
                     </ul>
```
Comment (Shaher-11) [inline index.html]: The languages should be styled as shown below
#### Original design
right
### Your Project
wrong

File: index.html [hunk 3]
```diff
@@ -8,30 +8,236 @@
...
+                        <h3 class="header">Framework</h3>
+                    </div>
+                    <ul class="container code-skill" id="frame-wrk-list">
+                        <li class="lang">Bootstrap</li>
+                        <li class="lang">Ruby on rails</li>
+                        <li class="lang">Rspec</li>
+                        <li class="lang">Copybare</li>
+                        <li class="lang">Selenium</li>
+                    </ul>
+                </div>
```
Comment (Shaher-11) [inline index.html]: The framework should be styled as shown below
### Original design
wrong

### Your Project
wrong

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @fobadara,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:

## Optional Suggestions💡
_The following point is highly recommended to improve your code quality, Please consider implementing it.

- Try to commit more often as it's a great habit to acquire.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 1 above)
Comment (fobadara) [reply index.html:160]: Ok, I have done that. Thank you

File: index.html (hunk 2 above)
Comment (fobadara) [reply index.html]: I have done this too. It actually changed after I fixed some conflict

File: index.html (hunk 3 above)
Comment (fobadara) [reply index.html]: Done, I will submit now

**Review by Shaher-11 — APPROVED** [review]
#### Hello @fobadara,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**